contribute/code.rst

```diff
@@ -0,0 +1,211 @@
+Code changes
+============
+
+This page helps you get your changes to Ubuntu Touch's code merged. To help you with this, we describe the things we look for in code, commits, and changesets. We also take a look at what to expect in the code review process.
```

Comment:
To help you with this -> To assist you with this

removes a double "help you" in this paragraph